EIPS/eip-4361.md

```diff
@@ -0,0 +1,295 @@
+---
+eip: 4361
+title: Sign-In with Ethereum. 
+description: Standardizes off-chain authentication for Ethereum accounts to establish sessions.
+author: Wayne Chang (@wyc), Brantly Millegan (@brantlymillegan), Nick Johnson (@Arachnid)
+resolution: <TBD>
```

Comment:
```suggestion
```
This is not an EIP frontmatter field.

Reply:
@MicahZoltu from https://eips.ethereum.org/EIPS/eip-1#eip-header-preamble
```
` * resolution:` a url pointing to the resolution of this EIP
```

Is there a more updated version you could point me to that reflects its omission? Thanks

Reply:
https://raw.githubusercontent.com/ethereum/EIPs/master/eip-template.md I'll get a PR out to fix EIP-1 to match.

Ignore NizamShah7, it is just a spam account.

Reply:
Removed in https://github.com/ethereum/EIPs/pull/4361/commits/0975d6c0d5a610dd8209d9911c715702c8600c62 , thanks for your advice! Feel free to reopen if there's anything else.